archaius-zookeeper/src/main/java/com/netflix/config/source/ZooKeeperConfigurationSource.java

```diff
@@ -223,6 +224,10 @@ synchronized void deleteZkProperty(String key) throws Exception {
     }
     
     public void close() {
-    	Closeables.closeQuietly(pathChildrenCache);
+        try {
+            Closeables.close(pathChildrenCache, true);
+        } catch (IOException exc) {
+            logger.warn("IOException should not have been thrown.", exc);
```

Comment:
That's reasonable. I'll update this patch.

Though I think the same behavior is inherently altered as it's going to SLF4J instead of JUL (and I'd be surprised if most users were importing a JUL-SLF4J bridge). But I'm sure nobody wants more JUL!